pnpm-stage0.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: pnpm-stage0
+  version: 8.7.4
+  epoch: 0
+  description: "Fast, disk space efficient package manager"
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - nodejs-18
+    provides:
+      - pnpm=8.7.4
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - build-base
+      - nodejs-18
+      - ca-certificates-bundle
+      - curl
+      - git
+      - npm
+      - gcc-12
```

Comment:
I'm surprised we need `gcc` to build pnpm!

Reply:
node-gyp depends on gcc , I just kept it here once we turn on the package tests , as it required for tests 

https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md?plain=1#L31